satellite/metainfo/metainfo.go

```diff
@@ -674,8 +674,13 @@ func (endpoint *Endpoint) GetBucket(ctx context.Context, req *pb.BucketGetReques
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
 
+	convBucket, err := convertBucketToProto(ctx, bucket)
```

Comment:
convBucket is a weird name. could you just call it bucket or protoBucket or pbBucket?